src/components/ReadMore.js

```diff
@@ -44,15 +70,23 @@ export const ReadMore = hh(class ReadMore extends Component {
     });
   };
 
+  getReadLink = (fun, text, classes) => {
+    if (this.state.inline) {
+      return a({ onClick: () => fun() }, [text])
+    } else {
+      return a({ onClick: () => fun(), style: this.state.readStyle }, [
+        text,
+        span({className: classes, style: {padding: '0 1rem'}, 'aria-hidden': 'true'})
+      ])
+    }
+  }
```

Comment:
I think this is another example where ternaries can help reduce code duplication:
```
    const linkElements = this.state.inline ? [text] :
      [text, span({className: classes, style: {padding: '0 1rem'}, 'aria-hidden': 'true'})];
    const linkElementsStyle = this.state.inline ? {} : this.state.readStyle;
    return a({ onClick: () => fun(), style: linkElementsStyle}, linkElements);
```

Reply:
For bonus points, here is a way to destructure two constants from a single ternary expression:
```
    const {linkElements, linkElementsStyle} = this.state.inline ?
      {linkElements: [text], linkElementsStyle: {}} :
      {
        linkElements: [
          text,
          span({
            className: classes,
            style: {padding: '0 1rem'},
            'aria-hidden': 'true',
          })], linkElementsStyle: this.state.readStyle,
      };
    return a({ onClick: () => fun(), style: linkElementsStyle}, linkElements);
```